bin/interactive-install.sh

```diff
@@ -28,41 +28,55 @@ echo
 echo -e "Creating namespace securecodebox-system"
 kubectl create namespace securecodebox-system || echo -e "Namespace already exists.. "
 
-if [[ $PWD == *"bin" ]]; then
-  cd ..
-fi
+[ -z "${SCRIPT_DIRECTORY:-}" ] \
+  && SCRIPT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )" \
+  && export SCRIPT_DIRECTORY
+
+cd "$SCRIPT_DIRECTORY"
+cd ..
```

Comment:
That's not very intuitive for an experiences shell script reade ;-)

A good patter is to define a base directory and work with that:

```bash
BASE_DIR=$(dirname "${SCRIPT_DIRECTORY}")
```

From now on always use `"${BASE_DIR}/path/to/my/stuff"` style paths. And always escape paths with quotes and expansion to prevent unwanted globbering or wordsplitting: `"${VAR}"`.